src/main/com/cqfn/degitx/simulator/Request.kt

```diff
@@ -2,10 +2,51 @@ package com.cqfn.degitx.simulator
 
 /**
  * Stub for Request sent via Network to Address.
- * Data class.
+ *
  * @param sender Initiator of request.
  * @param id Optional ID of request for cases when response is required. Default value is 0 = No Response Required.
  * @param size Size of body instead of real data. It's enough for Simulation.
  * @param timespent Time spent to process this request. Will go to final log for analysis. Increments all way.
  */
```

Comment:
There are some new params, please update comment and probably move it to constructor while it is no more a data class.

Reply:
Done.